.github/actions/upload_coverage/action.yml

```diff
@@ -0,0 +1,15 @@
+name: coverage
+inputs:
+  flags:
+    required: false
+  codecov_token:
+    required: true
+runs:
+  using: composite
+  steps:
+  - uses: codecov/codecov-action@v3
+    with:
+      flags: ${{ inputs.flags }}
+      token: ${{ inputs.codecov_token }}
```

Comment:
Is that always masked when we pass the input from secrets? We donot wanna leak that.

Reply:
Looks like so from ci output.